types/club.ts

```diff
@@ -0,0 +1,6 @@
+export interface Club {
+  name: string;
+  eng: string;
```

Comment:
변수명이 engName 정도로 좀 더 구체적이면 좋을 것 같아요. 

```suggestion
  engName: string;
```

Reply:
만약 바꾼다면 ClubTitle도!